run-train-and-inference.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+YAML_CONFIG=/configmount/config.yaml
+CONFIG_SETTING=full_field
+WANDB_UUID=$(python -c "import wandb; print(wandb.util.generate_id())")
+export WANDB_RUN_GROUP=fourcastnet-train-and-inference-$WANDB_UUID
+
+# run training
+export WANDB_JOB_TYPE=training
+torchrun --nproc_per_node 1 /opt/ERA5_wind/train.py --yaml_config $YAML_CONFIG --config $CONFIG_SETTING
+
+echo ===============================================================================
+echo ======================FINISHED TRAINING STARTING INFERENCE=====================
+echo ===============================================================================
+
+# run inference
+export WANDB_JOB_TYPE=inference
+#TODO(gideond) factor out /output
+python /opt/ERA5_wind/inference/inference.py --yaml_config $YAML_CONFIG --config $CONFIG_SETTING --override_dir /output --weights /output/$CONFIG_SETTING/00/training_checkpoints/best_ckpt.tar --vis
```

Comment:
```suggestion
python /opt/ERA5_wind/inference/inference.py --yaml_config $YAML_CONFIG --config $CONFIG_SETTING --vis
```
I think this should just work, because we are using the same config for the training and inference (i.e. the `expDir` param will be same).

See: 
https://github.com/ai2cm/FourCastNet/blob/b7d7f32a10cd09e25d3be2ea1d6967269033def4/inference/inference.py#L394-L400

and

https://github.com/ai2cm/FourCastNet/blob/b7d7f32a10cd09e25d3be2ea1d6967269033def4/inference/inference.py#L406